scarb/src/core/lockfile.rs

```diff
@@ -245,10 +251,7 @@ mod tests {
 
     #[test]
     fn empty() {
-        let lock = Lockfile {
-            version: Default::default(),
-            packages: Default::default(),
-        };
+        let lock = Lockfile::new(vec![].into_iter());
```

Comment:
```suggestion
        let lock = Lockfile::new([]);
```

This should be possible I believe if new takes `impl IntoIter`